feat(wsp): Retirer bouton pour agrandir description

Rotomeca opened the pull request:
>Enlever ce bouton et faire en sorte que l'on puisse cliquer sur la description comme un bouton pour ouvrir la description en grand

>(Le bouton n'apparaît pas lorsqu'il n'y a pas de description et ça porte à confusion)

FEAT: MANTIS 0008851

Rotomeca committed:
fix(wsp): Retirer bouton pour agrandir description

>Enlever ce bouton et faire en sorte que l'on puisse cliquer sur la description comme un bouton pour ouvrir la description en grand
>(Le bouton n'apparaît pas lorsqu'il n'y a pas de description et ça porte à confusion)

FIX: MANTIS 0008851

Rotomeca committed:
doc(wsp): Documentation du composant

Rotomeca committed:
fix(wsp): le css de la navbar n'était pas minifié

AGoubs reviewed (changes requested):
On peut ouvrir la description meme s'il n'y en a pas

Rotomeca committed:
fix(wsp): Ne pas pouvoir cliquer sur le bouton si il n'y a pas de descriptions

https://github.com/messagerie-melanie2/Roundcube-plugins-Mel/pull/96#pullrequestreview-2674705416

Rotomeca committed:
fix(app): ajout de `.js` manquant
